docs/metrics.md

```diff
@@ -1,37 +1,140 @@
 # Metrics
 
-> :construction: &nbsp;Status: Experimental - exported metric data and
-> configuration properties will change once OpenTelemetry metrics become available.
+> :construction: &nbsp;Status: Experimental
 
-The Splunk Distribution for OpenTelemetry Node.js gathers runtime metrics and allows to send custom metrics via a [SignalFx client](https://github.com/signalfx/signalfx-nodejs).
+The Splunk Distribution for OpenTelemetry Node.js configures the default OpenTelemetry meter provider and can collect
+runtime metrics.
 
 ## Configuration
 
 For configuration options, see [advanced configuration](advanced-config.md#metrics).
 
-## Runtime metrics
+## Usage (custom metrics)
 
-The following is a list of metrics automatically collected and exported:
+Add `@opentelemetry/api-metrics` to your dependencies.
+
+```javascript
+const { startMetrics } = require('@splunk/otel');
+const { Resource } = require('@opentelemetry/resources');
+const { metrics } = require('@opentelemetry/api-metrics');
+
+// All fields are optional.
+startMetrics({
+  // Takes preference over OTEL_SERVICE_NAME environment variable
+  serviceName: 'my-service',
+  // The suggested resource is filled in via OTEL_RESOURCE_ATTRIBUTES
+  resourceFactory: (suggestedResource: Resource) => {
```

Comment:
This is API that should (eventually) be mirrored to tracing as well.
I'd not force the user to know the `Resource` API to use it and just have it return an object that will be passed and merged to detected Resource.